src/modules/user/controllers.ts

```diff
@@ -31,10 +31,12 @@ class UserOperations {
    * @returns null
    */
   static async updateProfile(request: Request, response: Response) {
+    const data = await models.updateProfile(request.body)
```

Comment:
never pass daa to model without checking if it's valid or not. 
- check if it's undefined
- check if it contains the required properties (required:true in schema)

Reply:
I added the validation in the Models to keep it consistent with the registerUser method.